README.md

```diff
@@ -1,53 +1,51 @@
-## Project name
+## gcloud-golang-todos
 
-A description of what this project does and who it serves.
+> [TodoMVC](http://todomvc.com) backend using [gcloud-golang](//github.com/GoogleCloudPlatform/gcloud-golang).
 
-Include authorship, support contact and release information.
 
+### Prerequisites
 
-## Project setup, installation, and configuration
+1. Create a new cloud project on [console.developers.google.com](https://console.developers.google.com)
+1. Export your project id:
+    
+    ```sh
+    gcloud config set project <project id>
+    ```
 
-How do I, as a developer, start working on the project?
+1. go-get this code!
 
-1. What dependencies does it have (where are they expressed) and how do I install them?
-1. Can I see the project working before I change anything?
+    ```sh
+    go get -u github.com/GoogleCloudPlatform/gcloud-golang-todos
+    ```
 
+1. Initialize the todomvc subproject:
```

Comment:
I decided it basically made sense to have TodoMVC as a submodule, since we're really just serving their static content. That said, git is a little weird for this stuff, and doesn't fetch it by default when cloning the encompassing repo.